CHANGELOG.md

```diff
@@ -100,6 +100,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   profile. #852, #876, #883, #896, #903
 - During node discovery, prefer the primary network interface, if defined. #775
 
+- default is now protected and can't be deleted
+- first container imported container is added to the default profile
```

Comment:
If this is using the host shim now, can all the special-casing around the default profile and its container be removed?